provisioning/transport/http/src/SymmetricKeyCredentials.cs

```diff
@@ -12,30 +12,11 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Transport
 {
     internal class SymmetricKeyCredentials : ServiceClientCredentials
     {
-        private const string SASHeaderName = "SharedAccessSignature";
-        private const string Registration = "registration";
         private readonly string SymmetricKey;
-        private volatile string _sasToken;
 
         public SymmetricKeyCredentials(string symmetricKey) : base()
         {
             SymmetricKey = symmetricKey;
         }
-
-        public override Task ProcessHttpRequestAsync(HttpRequestMessage request, CancellationToken cancellationToken)
```

Comment:
I'm not sure what the bar is for removing code in this PR. Is it simply that a member doesn't have any references? I'm worried we're going too far in some cases.